### ISSUE TITLE
Investigate email Send rate slow down

### ISSUE THREAD

**Issue by willeybryan** [cds-snc/notification-api#1060]
**Steps to Reproduce**
- Send 10k Emails
- Previous Send rate of 50 emails a second means it should be done in less than 4 minutes. 
- it Took 4 hours for this service https://notification.alpha.canada.ca/services/e7825299-cf7b-4272-aaa8-89ca5c9913cd

**Expected**
~5 minutes

**Actually**
Reportedly 4 hours

**Comment by brdunfield** [comment]
Do we have any more info on this?
Does "reportedly 4 hours" mean that they're telling us it took that long, or we've verified it did? Were the emails stuck in "Sending" the whole time?

Just trying to see if this was us or SES.

**Comment by willeybryan** [comment]
@brdunfield 
They did another bulk send over lunch and I tracked it. Seems the Issue seems to no longer be happening. It last happened on Friday, but today the bulk sends from the service are working as expected. 

It may have been SES throttling because their failure rate was so high. I gave them instructions on how to reduce it and so far it seems to be working. 

I will call this a false alarm and done for now, thanks for your time.
